.travis.yml

```diff
@@ -32,10 +33,16 @@ addons:
   apt:
     packages:
       - swig
+  firefox: latest
 
 before_install:
   - scripts/travis_es.sh
   - /tmp/elasticsearch/elasticsearch-1.6.2/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
+  - wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.13.0/geckodriver-v0.13.0-linux64.tar.gz
```

Comment:
We now have 0.14.0, but again, I question why we're adding this to Travis rather than simply restoring the tests in CircleCI?